build/post-configure.sh.in

```diff
@@ -31,7 +31,6 @@ echo ""
 echo "C compiler...................................... : @CC@"
 echo "C compiler vendor............................... : @CC_VENDOR@"
 echo "Fortran compiler................................ : @FC@"
-echo "Fortran compiler vendor......................... : @FC_VENDOR@"
```

Comment:
This was always saying "@FC_VENDOR@" when running the config script. Removing since unused